doc/en/fixture.rst

```diff
@@ -408,31 +577,170 @@ containers for different environments. See the example below.
     def docker_container():
         yield spawn_container()
 
+.. _`fixture order`:
+
+Fixture instantiation order
+---------------------------
+
+When pytest wants to execute a test, once it knows what fixtures will be
+executed, it has to figure out the order they'll be executed in. To do this, it
+considers 3 factors:
+
+1. scope
+2. dependencies
+3. autouse
```

Comment:
Here again, autouse hasn't been described yet (I see in this case this is a preexisting issue)

Reply:
I mentioned it above, but perhaps the description of what autouse fixtures are should be moved up?